src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
it's forbidden to use anything except `box-sizing` in `*`
reset padding and margins should be in the `body`